src/Components/UI/button/MyButton.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import style from './MyButton.module.css'
+
+const MyButton = ({children, className, ...props}) => {
```

Comment:
as we have here unprotected props passing, this component is especially requires type check, meaning [PropTypes](https://www.npmjs.com/package/prop-types), which actually was requested by the task, by the way, so please consider adding it